utils/general.py

```diff
@@ -36,6 +38,69 @@
 # Prevent OpenCV from multithreading (to use PyTorch DataLoader)
 cv2.setNumThreads(0)
 
+# Download objects
+def download_file_blob_to_dir(directory, file_blob):
+    file_name = file_blob.name.split("/")[2]
+    destination = Path(directory) / f"{file_name}"
+    destination.touch()
+    blob.download_to_filename(destination)
+
+
+# Place GCS objects in their respective local directories
+def place_object(local_directory, file_blob):
+
+    train_image_directory = local_directory[0]
+    train_label_directory = local_directory[1]
+    validation_image_directory = local_directory[2]
+    validation_label_directory = local_directory[3]
+
+    if file_blob.name.startswith("images/t") and file_blob.name.endswith(".jpg"):
```

Comment:
For the sake of completion, we should spell out the name in whole. This way a reader isn't wondering what the `t` stands for.
```suggestion
    if file_blob.name.startswith("images/train") and file_blob.name.endswith(".jpg"):
```

Reply:
Fixed 211c0f659f2277141daab4bd1c64843cdd2b50e4

Reply:
Fixed 211c0f659f2277141daab4bd1c64843cdd2b50e4